CONTRIBUTING.MD

```diff
@@ -0,0 +1,164 @@
+# Contributing to Chef's knife-openstack
```

Comment:
You're probably better off just linking to the main chef contributing doc. When we switched over to the DCO process we removed most of the individual docs since they had all diverged and the maint had become burdensome